docs/en/enterprise-edition/content-collections/runtime-security/agentless-scanning/configure-accounts/configure-gcp.adoc

```diff
@@ -18,45 +18,67 @@ Complete the steps below to configure agentless scanning based on the needs of y
 . Click the edit button of your cloud account.
 
 . Go to the *Agentless Scanning* section.
++
+image::agentless-gcp-pcee-advanced-settings.png[]
 
-. *Hub account*: Enable the toggle to configure this account as a xref:../agentless-scanning-modes.adoc[hub account].
+. Expand the *Advanced settings* and provide the appropriate values based on your deployment for the following items.
 
-. Expand the *Advanced settings*.
-+
-image::runtime-security/agentless-gcp-configuration.png
+. Click *Save* to return to *Compute > Manage > Cloud accounts*.
+
+[#gcp-agentless-modes]
+==== Scan Settings
 
-.. *Select where to scan*: For GCP accounts, you can decide between xref:../agentless-scanning.adoc#scanning-modes[two scanning modes].
+*Select where to scan*: For GCP accounts, you can decide between xref:../agentless-scanning.adoc#scanning-modes[two scanning modes].
 
-... *Same Account*: Perform the agentless scanning process using this account.
+* *Same Account*: Perform the agentless scanning process using this account.
 
-... *Hub Account*: Perform the agentless scanning process using a centralized hub account.
+* *Hub Account*: Perform the agentless scanning process using a centralized hub account.
 Select another account from the list to use as the centralized hub account to scan this account.
- on the target accounts.
+on the target accounts.
```

Comment:
what is this? please remove - i see this in all other CSPs as well

Reply:
Done. It was a remanant from a previous version.